notebooks/proj_sys_eng/sitcom_integration/LVV-T2668-Measure_torque_capacity_of_the_CCW_drive.ipynb

```diff
@@ -0,0 +1,1057 @@
+{
```

Comment:
<p>"camera cable wrap" --&gt; " camera cable wrap (CCW)"</p><p> </p><p>But actually, you used the word of CCW before having the full name here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>